.github/actions/spelling/patterns.txt

```diff
@@ -1,21 +1,9 @@
-[Ll][Oo][Mm][Bb][Ii][Qq]
 (\((.*?)\))
 (https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]+\.[^\s]{2,}|www\.[a-zA-Z0-9]+\.[^\s]{2,})
 [Hh]ome[Pp]age
-[Dd]otnet
-[Ll][Ii][Nn][Qq]
-[Ll]ocalhost
-[Mm][Vv][Cc]
-[Uu]ncomment
-[Jj]ira
-[Ii][Ii][Ss]
-[Cc]mdlet
-[Aa]tata
-[Cc][Ll][Oo][Uu][Dd][Ss][Mm][Ii][Tt][Hh]
-[Ss][Dd][Kk]
-[Rr][Ee][Aa][Dd][Mm][Ee]
-[Oo][Ss][Oo][Cc][Ee]
-[Ss][Ff][Cc]
+[Cc](loud|LOUD)[Ss](mith|MITH)
 [Xx][Uu]nit
-[Xx][Pp][Aa][Tt][Hh]
-[Rr]eadme
+[Xx][Pp]ath
+[{]?[0-9a-zA-Z]{8}-([0-9a-zA-Z]{4}-){3}[0-9a-zA-Z]{12}[}]?
+[s][h][a][0-9]{3}-[0-9a-zA-Z\+\/]{64}
```

Comment:
This is equivalent to:
```suggestion
sha[0-9]{3}-[0-9a-zA-Z+/]{64}
```
No need to escape characters in character classes (enclosed by `[` and `]`).